inductiva/tasks/task.py

```diff
@@ -198,7 +200,7 @@ def get_output_files_info(self):
     def download_outputs(
         self,
         filenames: Optional[List[str]] = None,
-        output_dir: Optional[pathlib.Path] = None,
+        output_dir: Optional[types.Path] = None,
         uncompress: bool = True,
         rm_archive: bool = True,
```

Comment:
just out of curiosity. is this removing the *remote* archive, i.e. the one on the bucket?
If so, please add "remote" to this flag, otherwise it is ambiguous.

Reply:
no, from the code: 
`  if rm_archive:
        zip_path.unlink()`
which means that this is just removing the zip archive.

Reply:
but that feature makes sense.

Reply:
ok. so that should be a new feature.
But still, the naming is ambiguous. What name would you suggest to make this variable less ambiguous?

In general, you should try to spot this sort of ambiguities. Or, when someone asks a stupid question like i did, then you should try to see if this question was because something is not clear and you should try to correct the source of ambiguity.

Reply:
I think that rm_zip_archive can be a better name

Reply:
Then, you should do that. If you identify a source of ambiguity, then just fix it.

Also, you could even be more explicit.

rm_downloaded_zip_archive